packages/flutter/lib/src/material/input_border.dart

```diff
@@ -333,15 +333,15 @@ class OutlineInputBorder extends InputBorder {
   ///
   /// The corner radii must be circular, i.e. their [Radius.x] and [Radius.y]
   /// values must be the same.
-  final BorderRadius borderRadius;
+  final BorderRadiusGeometry borderRadius;
```

Comment:
@HansMuller It looks like this change breaks the flutter_gallery customer test, so it's probably a larger breaking change than I thought.

However, long term this seems like the right change to make: other borders are direction sensitive, and it makes sense for this one to also be direction sensitive. Should I go ahead and try and land this?  I don't think it's something that can be addressed with an automatic fix, since sometimes you need access to the TextDirection to resolve the conversion.

(If I do try and land this, I'll separate it into its own change)

Reply:
I'm going to revert this part of the change for now.